crates/turborepo-ui/src/tui/app.rs

```diff
@@ -118,10 +282,15 @@ pub fn run_app(tasks: Vec<String>, receiver: AppReceiver) -> Result<(), Error> {
     let ratio_pane_width = (f32::from(size.width) * PANE_SIZE_RATIO) as u16;
     let full_task_width = size.width.saturating_sub(task_width_hint);
 
-    let mut app: App<Box<dyn io::Write + Send>> =
-        App::new(size.height, full_task_width.max(ratio_pane_width), tasks);
+    let mut app: App<Box<dyn io::Write + Send>> = App::new(size.height, size.width, tasks);
```

Comment:
I don't believe this was an intentional change.
```suggestion
    let mut app: App<Box<dyn io::Write + Send>> = App::new(size.height, full_task_width.max(ratio_pane_width), tasks);
```